togetherly/lib/services/person_service.dart

```diff
@@ -89,7 +92,7 @@ class PersonService {
       );
 
   Map<String, dynamic> _parentToMap(Parent parent) => {
-        'id': parent.id,
+        // 'id': parent.id,
```

Comment:
It would be better to delete this line and line 77, rather than just comment them out. You were correct to at least disable them.

Reply:
Done